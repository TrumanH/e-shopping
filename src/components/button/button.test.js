import renderer from 'react-test-renderer';
import Button, { BUTTON_TYPES }  from './button.component';

it('changes the class when hovered', () => {
  const component = renderer.create(
    <Button buttonType={BUTTON_TYPES.inverted}>Test</Button>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
  }
);