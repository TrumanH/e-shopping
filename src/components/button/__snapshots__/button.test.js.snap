// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes the class when hovered 1`] = `
<button
  className="sc-dkrFOg sc-eDvSVe bAenDU iJPJvM"
>
  Test
</button>
`;
